gnmi_cli/README.md

```diff
@@ -0,0 +1,21 @@
+The GNMI CLI may be used to demo the `onos-config`. Example files in this directory.
+
+See more details at [https://docs.onosproject.org/onos-config/docs/gnmi/](https://docs.onosproject.org/onos-config/docs/gnmi/)
+
+Typical usage for **get** is as follows:
+```
+gnmi_cli -get -address localhost:5150 \
+    -timeout 5s -en PROTO -alsologtostderr -insecure \
```

Comment:
-en option is just available for gnmi_cli that we customized. Perhaps you should mention it somewhere if it is not mentioned.